theano/tensor/tests/test_opt.py

```diff
@@ -5675,18 +5675,19 @@ def test_local_useless_split():
 
 def test_local_flatten_lift():
     for i in xrange(1, 4):
-        op = tensor.Flatten(i)
         x = tensor.tensor4()
-        out = op(T.exp(x))
+        out = tensor.flatten(T.exp(x), i)
         assert out.ndim == i
         mode = compile.mode.get_default_mode()
         mode = mode.including('local_flatten_lift')
```

Comment:
Nah, I don't think so. It's just that the FAST_COMPILE mode doesn't run the optimization unless we force it into config.mode. You can see that it is added in [test_local_reshape_lift](https://github.com/Theano/Theano/blob/master/theano/tensor/tests/test_opt.py#L5733-L5735)